free-2.md

```diff
@@ -46,6 +92,16 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+To the best of my knowledge, the best that we have currently are iPhone/android
+apps. The most popular of these is https://itunes.apple.com/us/app/clue-period-tracker/id657189652?mt=8
+which does a good job of staying gender neutral and providing a lot of scientific access.
+To be honest, this app does a great job of providing information, reports, and calculations.For
+example, they have lots of in app reference data about what typical symptoms look like. You 
+can also produce data reports based on all the information you put in (about periods, sleep,
+emotional state, etc). In comparison, a lot of other period apps are extremely feminized,
+which is exclusive to people with periods who don't identify as female (see http://www.makeuseof.com/tag/4-best-period-tracker-apps-ios-android/). However, it is still 
+limited to people who have access to smartphones/devices. For thosein lower income situations, 
+it's easier to have access to a computer than a smartphone.
 
```

Comment:
I'm not sure I agree with you last sentence. It seems like an App addresses the problem very well because people constantly have their phones on them so they can keep the information up to date, but if they have to do it on the computer they won't necessarily be adding updates as frequently, and especially for lower income situations, people may not use a computer every day so it might be harder to input enough data to get useful information from it.